elastic/apm-server
# Change cgroup "id" fields to keyword

## Motivation/summary

Update `system.process.cgroup.cpu.id` and `system.process.cgroup.cpuacct.id` to have keyword type, matching beats.

## How to test these changes

Run `apm-server export template`, check the field types are "keyword" for `system.process.cgroup.cpu.id` and `system.process.cgroup.cpuacct.id`.

## Related issues

Closes https://github.com/elastic/apm-server/issues/5175
